apps/contrib/templates/meinberlin_contrib/includes/inline_form.html

```diff
@@ -0,0 +1,23 @@
+{% load i18n %}
+<div class="formset__form{% if form.name.errors %} formset__form--has-error{% endif %}"
+     data-container="body"
+     data-toggle="popover"
+     data-trigger="hover"
+     data-placement="top"
+     data-content="{% for error in form.name.errors %}{{ error }}{% endfor %}">
```

Comment:
Ok, I tested that now. There seem to be some issues:

- Not accessible via keyboard
- The error is not counted in the tab